docs/AboutUs.md

```diff
@@ -33,12 +33,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Johnny Doe
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/choiwab.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/choiwab)] [[portfolio](https://jaechoi.vercel.app/)]
```

Comment:
Please add a separate `choiwab.md` file to list your contributions just for this project.